demos/text_detection_demo/main.cpp

```diff
@@ -33,15 +33,17 @@
 
 using namespace InferenceEngine;
 
-
-std::vector<cv::Point2f> floatPointsFromRotatedRect(const cv::RotatedRect &rect);
-std::vector<cv::Point> boundedIntPointsFromRotatedRect(const cv::RotatedRect &rect, const cv::Size& image_size);
-cv::Point topLeftPoint(const std::vector<cv::Point2f> & points, int *idx);
-cv::Mat cropImage(const cv::Mat &image, const std::vector<cv::Point2f> &points, const cv::Size& target_size, int top_left_point_idx);
-void setLabel(cv::Mat& im, const std::string& label, const cv::Point & p);
+std::vector<cv::Point2f> floatPointsFromRotatedRect(
+    const cv::RotatedRect &rect);
+std::vector<cv::Point> boundedIntPointsFromRotatedRect(
+    const cv::RotatedRect &rect, const cv::Size &image_size);
+cv::Point topLeftPoint(const std::vector<cv::Point2f> &points, int *idx);
+cv::Mat cropImage(const cv::Mat &image, const std::vector<cv::Point2f> &points,
+                  const cv::Size &target_size, int top_left_point_idx);
+void setLabel(cv::Mat &im, const std::string &label, const cv::Point &p);
```

Comment:
Please don't reformat unrelated code. I can barely tell what you actually changed.